addons/website_sale_delivery/static/src/js/website_free_delivery_tour.js

```diff
@@ -0,0 +1,60 @@
+odoo.define('website_sale_delivery.tour', function (require) {
+    'use strict';
+
+    var tour = require("web_tour.tour");
+    var rpc = require("web.rpc");
+
+    tour.register('check_free_delivery', {
+            test: true,
+            url: '/shop',
+        },
+        [
+            {
+                content: "Set fixed price for delivery charges",
+                trigger: '#oe_main_menu_navbar',
+                run: function () {
+                    return rpc.query({
+                        'model': 'delivery.carrier',
+                        'method': 'write',
+                        'args': [[1], {
+                            'fixed_price': 2,
+                            'free_over': true,
+                            'amount': 10
+                        }],
+                    })
+                }
+            },
+            {
+                content: "search conference chair",
```

Comment:
As the search is [already tested in other tours](https://github.com/odoo/odoo/blame/04b2b9314354b9e009214db0a4ecc83096ac6fea/addons/website_sale/static/src/js/website_sale_tour_buy.js#L15), could you replace the 2 steps that search for the chair by starting directly the tour on the search result page like [in this tour](https://github.com/odoo/odoo/blame/04b2b9314354b9e009214db0a4ecc83096ac6fea/addons/website_sale/static/src/js/website_sale_tour_shop_custom_attribute_value.js#L10)? That will speed up the tour :+1:

Reply:
jumping now directly to `/shop?search=conference chair`